frontend/layouts/default/index.tsx

```diff
@@ -87,87 +88,6 @@ const DefaultLayoutComponent = function ({
             apiVersion="2020-01"
          >
             <Box>
-               <Head>
-                  <title>{title || 'iFixit'}</title>
-                  <link
-                     rel="apple-touch-icon"
-                     sizes="57x57"
-                     href="https://assets.cdn.ifixit.com/static/icons/ifixit/apple-touch-icon-57x57.png"
-                  />
```

Comment:
Will this remove these tags from all pages that use the `pages` router?

Reply:
I was wondering this too, but did not check. It looks like this does remove the icons.

https://react-commerce-prod-git-hello-world-app-router-ifixit.vercel.app/products/spudger
https://react-commerce-prod.vercel.app/products/spudger

<details>
<summary>Branch</summary>

![image](https://github.com/iFixit/react-commerce/assets/857362/ef558d8a-3637-4130-8f9a-5034926a598b)

</details>

<details>
<summary>Production</summary>

![image](https://github.com/iFixit/react-commerce/assets/857362/af91e6c2-2be8-4ce2-89de-1dcf7529890f)

</details>

@federicobadini I assume this was not intended?

Due to the time difference, I am going to push a commit to add these links back, but let me know if this is incorrect

Reply:
This is tricky, because now we might be double-including these tags in `frontend/app/layout.tsx`. Though maybe the `<Head>` component just doesn't work/render anything in the app router?